src/esp/bindings_js/modules/test_page.js

```diff
@@ -75,6 +79,41 @@ class TestPage {
       this.expect(trans.y() < dropPos.y(), "trans.y() < dropPos.y()");
     }
 
+    console.log("Testing raycast");
```

Comment:
The other tests in this function are roughly one line. Since your test is many lines, let's refactor it into a separate helper function that you call here.

Reply:
Done, thanks for the suggestion!